src/Common/src/Interop/Unix/libssl/Interop.SafeSslHandle.cs

```diff
@@ -18,6 +18,12 @@ public SafeSslContextHandle(IntPtr method)
                 handle = SSL_CTX_new(method);
             }
 
+
+            /// Use this constructor only with valid ssl_ctx pointer.
```

Comment:
Scrolling down, it looks like by removing the public FromSsl.. method and returning SafeHandle from SSL_get_SSL_CTX should take care of this